ops/terraform/.gitignore

```diff
@@ -1,3 +1,4 @@
 *.backup
 *.tfvars
-.terraform
\ No newline at end of file
+.terraform
```

Comment:
I think you can remove this line now, given the root `.gitignore` I added last week.